catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/query/GazetteerQueryCatalog.java

```diff
@@ -99,6 +102,8 @@
 
   private List<Filter> featureCodeFilters;
 
+  private String geocoderCollection = "suggester";
```

Comment:
:pencil2: I could see us having other suggesters in the future.  Maybe `gazetteer` would be a better collection name.